scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -638,6 +640,14 @@ def get_image_name_for_promotion(self, tag: str = None) -> str:
         if tag:
             image += f":{tag}"
         return image
+    
+    def get_dockerfile_contents(self) -> str:
+        """Dockerfile contents"""
+        dockerfile = ""
+        with open(self.dockerfile_with_path, "r") as f:
+            dockerfile = f.read()
+
+        return dockerfile

```

Comment:
Consider simplifying:
```suggestion
        with open(self.dockerfile_with_path, "r") as f:
            return f.read()
```